spec/support/dwp_setup.rb

```diff
@@ -0,0 +1,25 @@
+module DwpSetup
+
+  def build_with_bad_requests(yes_response = 5, bad_requests = 5)
```

Comment:
I'd maybe consider prefixing these methods with `dwp_checker` because in the code they're used without the module. But it's not a big issue.